pkg/config/config.go

```diff
@@ -165,22 +169,16 @@ type Config struct {
 	ResultsDir  string `json:"ResultsDir" mapstructure:"ResultsDir"`
 
 	///////////////////////////////////////////////
-	// Data collection options
+	// Query options
 	///////////////////////////////////////////////
-	Resources []string `json:"Resources" mapstructure:"Resources"`
+	Resources []string      `json:"Resources" mapstructure:"Resources"`
+	Filters   FilterOptions `json:"Filters" mapstructure:"Filters"`
+	Limits    LimitConfig   `json:"Limits" mapstructure:"Limits"`
+	QPS       float32       `json:"QPS,omitempty" mapstructure:"QPS"`
+	Burst     int           `json:"Burst,omitempty" mapstructure:"Burst"`
```

Comment:
Slight nit to aggregate things here but it makes it easier to grok what the options are and how they're used.